app/notify_admin.go

```diff
@@ -20,15 +20,14 @@ import (
 
 const lastTrialNotificationTimeStamp = "LAST_TRIAL_NOTIFICATION_TIMESTAMP"
 const lastUpgradeNotificationTimeStamp = "LAST_UPGRADE_NOTIFICATION_TIMESTAMP"
-const defaultNotifyAdminCoolOffDays = 0.0104166667 // this is a temp change
+const defaultNotifyAdminCoolOffDays = 14
```

Comment:
We should probably consider moving this as a configuration setting so we can override for testing in the future

Reply:
I can create a ticket to track this, should I assign this to our team?

Reply:
You can send it over to Self-serve as the owner of Notify Admin feature

Reply:
https://mattermost.atlassian.net/browse/MM-50425